### ISSUE TITLE
[Forge 2768] [MC-1.12.2] TameableEntity Angelfish - Crash

### ISSUE THREAD

**Issue by Excaliburning** [DrZhark/mocreatures#103]
Hi !

Using the latest version for **Mo' Creatures 12.0.5 and Forge 2768**.
I've installed both your coremod and MoC on a friend's server. Nothing crashes until we actually log-into the server, which instantly makes us crash when we first render the world.
The presence of an "**Angelfish**" seems to be the issue, here's the [Crash Log](https://gist.github.com/Excaliburning/001c455a0969c70e78ae0393c67cc370).

It seems your CurseForge page isn't updated and I couldn't find the github page to fill in a proper report, so I posted to your [forums too](https://www.mocreatures.org/forum/m/13290502/viewthread/32291613-moc-client-crash-upon-joining-server-angelfish/page/1). I'll close the forum thread and link to here.

Thanks for your work so far, we've been using Mo' Creatures for years and I hope this little bad guy won't be the end of us !

**Comment by korlandril** [comment]
Hi, did you ever get to the bottom of this? it doesn't seem like anyone has been looking at this Github or the forums and I've had a similar issue for my own modpack and server with a variety of different entities from Mo' Creatures being the cause. I've just had to take it out, no other choice :/

**Comment by Excaliburning** [comment]
> 
> 
> Hi, did you ever get to the bottom of this? it doesn't seem like anyone has been looking at this Github or the forums and I've had a similar issue for my own modpack and server with a variety of different entities from Mo' Creatures being the cause. I've just had to take it out, no other choice :/

No, sadly. Haven't found a solution and as you said, no one seems to look at the issue tracker as well as the forums. So, here we are, with a mod we love but can't use.
I had to take it out as well, unsurprisingly our server works. I tried adding it again, it crashed again so we stopped at that.

edit: maybe disabling any mobs that makes you crash would (kinda) solve it for now, I hadn't the time to look into it back then.

**Comment by fizzy6868** [comment]
have the same issue i dont see why this is not working

**Comment by Excaliburning** [comment]
No idea.
No one seems to take care of the GitHub anymore, the forums are deserted as well, no update since October. The author didn't make any announcement about retiring but one can be sure that he's not been around in months by now.

**Comment by fizzy6868** [comment]
To fix the issue in 1.12.2 make sure you install the custom spawner off his website. But then you get another issue of nothing spawning
